src/renderer/src/lib.rs

```diff
@@ -12,15 +12,14 @@ extern crate mopa;
 
 extern crate glutin;
 extern crate cgmath;
-
-extern crate amethyst_ecs;
+extern crate specs;
 
 /// Contains the included Render Targets
 pub mod target;
 /// Contains the included Passes
 pub mod pass;
 
-use amethyst_ecs::{Component, VecStorage};
+use self::specs::{Component, VecStorage};
```

Comment:
@Emilgardis Indeed, implementing `DerefMut` would add some syntactic sugar that makes the code look a little prettier. However, it does not resolve the core problem that we are pulling in `specs` as a dependency of `amethyst_renderer` when it really doesn't need to be. The only reason the renderer is being kept outside the top-level crate is because it's intended to be usable *outside of amethyst and specs*. But that's out of the scope of this PR, so I'll open a new issue about it.